### PR TITLE
Add tables to LoadMuonNexus v2 documention to explain loading

### LINKED ISSUE #15829: Document LoadMuonNexus with a Table
Add to the LoadMuonNexus documentation a table explain where various items are read from the Nexus and put into the Workspace.

This is part of the work for issue #14940 .

### PR DESCRIPTION
**Pull request by KarlPalmen**
Adding tables to documentation of LoadMuonNexus version 2 to explain what items in the Nexus file go where in the workspace, when the algorithm is executed.

Check that the documentation is a good explanation of what LoadMuonNexus version 2 does, particularly for Nexus files that are used in the unit test.

Fixes #15829 .

Release notes will be updated when #14940 is done.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the coding standards? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [x] Has the relevant documentation been added/updated?
- [x] Is user-facing documentation written in a user-friendly manner?
- [x] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/algorithms/LoadMuonNexus-v2.rst [hunk 1]
```diff
@@ -39,32 +48,79 @@ Errors
...
++----------------------------------+--------------------------------------------+----------------------------------------+
+| Instrument                       | group ``Instrument``                       | Workspace instrument                   |
+|                                  |                                            | as loaded by LoadInstrumentFromNexus   |
++----------------------------------+--------------------------------------------+----------------------------------------+ 
+| Spectrum of each workspace index | [DET]``/spectrum_index``                   | Spectra-Detector mapping of workspace  |
++----------------------------------+--------------------------------------------+----------------------------------------+ 
+| Title (optional)                 | ``title``                                  | ``title`` in workspace                 |
++----------------------------------+--------------------------------------------+----------------------------------------+
+| Note orcomment (optional)        | ``note``                                   | ``comment`` in workspace               |
```
Comment (tom-perkins) [inline docs/source/algorithms/LoadMuonNexus-v2.rst]: space between "or" and "comment" (very minor point!)

**Comment by tom-perkins** [comment]
The new tables add a lot more information about the data loaded from v2 muon NeXus files, which is a very useful improvement. There is an explanation of how the algorithm tells v1 from v2 files and the documentation covers everything loaded from the unit test files.

However, the table of algorithm properties (which is not in the rst file but is auto-generated from `LoadMuonNexus.cpp`) is the same for the v1 and v2 algorithms. Perhaps a "v1 only" note could be added to the properties that are not loaded by v2:
- DeadTimeTable
- DetectorGroupingTable
- AutoGroup 
- MainFieldDirection

**Comment by KarlPalmen** [comment]
I see that it runs ``Algorithm2->copyPropertiesFrom( *Algorithm1 )`` to transfer properties. This could reduce code in many a ``RunChildAlgorithm`` function, but then it may only work if ``Algorithm2`` has exactly the same properties as ``Algorithm1``.

**Comment by tom-perkins** [comment]
The new changes now explain which properties are v1 only, and add to the changes already reviewed above. I can't think of anything else that is missing from the documentation now, and the RHEL6 build failure is unrelated to this PR, so happy to :shipit:!

**Comment by peterfpeterson** [comment]
@tom-perkins didn't tick any of the boxes. :scream:

**Pull request merged**